docs/byoh-instance-pre-requisites.md

```diff
@@ -4,6 +4,7 @@ The following pre-requisites must be fulfilled in order to add a Windows BYOH no
 * The Docker container runtime must be installed on the instance.
 * The instance must be on the same network as the Linux worker nodes in the cluster.
 * Port 22 must be open and running [an SSH server](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
+  * The default shell for the SSH server must be cmd. [Configuring the default shell for OpenSSH](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_server_configuration#configuring-the-default-shell-for-openssh-in-windows).
```

Comment:
s/cmd/Windows Command shell (cmd.exe)

Reply:
Perhaps make the _Windows Command shell_ text the link? Currently, the second sentence is just the linked URL title.

"The default shell for the SSH server must be the [Windows Command shell (cmd.exe)](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_server_configuration#configuring-the-default-shell-for-openssh-in-windows)."

Reply:
I havent found any Microsoft docs referring to cmd as `Windows Command shell`, I'm changing it to `cmd.exe` as thats what its most commonly referred to. I also see it being referred to `Command Prompt`, but to a much lesser extent.